scripts/local_dev.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -ex
+
+MNT_PATH="database"
+PLUGIN_NAME="vault-plugin-database-elasticsearch"
+PLUGIN_CATALOG_NAME="vault-plugin-database-elasticsearch"
+
+#
+# Helper script for local development. Automatically builds and registers the
+# plugin. Requires `vault` is installed and available on $PATH.
+#
+
+# Get the right dir
+DIR="$(cd "$(dirname "$(readlink "$0")")" && pwd)"
+
+echo "==> Starting dev"
+
+echo "--> Scratch dir"
+echo "    Creating"
+SCRATCH="$DIR/tmp"
+mkdir -p "$SCRATCH/plugins"
+
+echo "--> Vault server"
+echo "    Writing config"
+tee "$SCRATCH/vault.hcl" > /dev/null <<EOF
+plugin_directory = "$SCRATCH/plugins"
+EOF
+
+echo "    Envvars"
+export VAULT_DEV_ROOT_TOKEN_ID="root"
+export VAULT_ADDR="http://127.0.0.1:8200"
+
+echo "    Starting"
+vault server \
+  -dev \
+  -log-level="debug" \
+  -config="$SCRATCH/vault.hcl" \
+  -dev-ha -dev-transactional -dev-root-token-id=root \
+  &
+sleep 2
+VAULT_PID=$!
```

Comment:
Shouldn't this be one line up? Or will it ignore a PID from `sleep 2`?

Reply:
Yeah, `$!` is the pid of the last background process, so it ignores the `sleep 2`. Doesn't hurt to move it up though.